src/perms.ts

```diff
@@ -12,7 +12,10 @@ export const KEY_READ_ACTIONS = [
 ];
 
 export const WRITE_DATA_ACTIONS = [
+  //'dynamodb:BatchWriteItem',
   'dynamodb:PutItem',
+  //'dynamodb:UpdateItem',
+  //'dynamodb:DeleteItem',
```

Comment:
Were these changes intentional?

Reply:
Yes, to only give putitem permission as minimum permission.